### ISSUE TITLE
[Screen reader - Azure Iot explorer  - Refresh] Data in the rows is not associated with the column headers

### ISSUE THREAD

**Issue by Amit8527510735** [OfficeDev/office-ui-fabric-react#11652]
**User Experience:**
If data in the rows is not associated with the column header, the screen reader will get confused regarding the information of the data present in the rows.

**Pre-requisites:** Turn on screen reader

**Repro Steps:**
1. Launch 'Azure IoT explorer' app from start menu.
2. 'App Configurations' pop up appears, navigate all controls of it.
3. Enter valid connection string and select connect button.
4. 'Azure IoT explorer'  home page launched, navigate 'Refresh' button and select it.
5. Navigate to the data rows present in the table.
6. Verify whether the data in the rows is associated with the column headers or not.

**Actual Result:**
In narrator scan mode, Data in the rows is not associated with the column headers. Screen reader is announcing it as ‘link 123’, ‘Enable’ and so on.
 
**Expected Result:**
In narrator scan mode, Data in the rows should be associated with the column headers. Screen reader should announce the information as ‘Device ID link 123’, ‘Status Enabled’ and so on.
 
**Note:**
The same issue exists with JAWS screen reader also.

**MAS Reference:**
[MAS 1.3.1 - Info and Relationships (40)](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7b54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4%7d)
![MAS1 3 1_Data in rows is not associated with the column headers](https://user-images.githubusercontent.com/53263029/72061357-b6023700-32fb-11ea-990f-9d279ea5c3c9.png)

Bug copied from Azure IOT Explorer Bug id:-[Bug 5811234](https://msazure.visualstudio.com/One/_workitems/edit/5811234)
